conformance/smart-config-fhirr4-service/README.md

```diff
@@ -1 +1,134 @@
+    scopesSupported = [
+        "openid",
+        "fhirUser",
+        "launch",
+        "launch/patient",
+        "patient/*.cruds",
+        "user/*.cruds",
+        "offline_access",
+    ]
+    responseTypesSupported = [
+        "code",
+        "id_token",
+        "token",
+        "device",
+        "id_token token"
+    ]
+    capabilities = [
+        "launch-ehr",
+        "launch-standalone",
+        "client-public",
+        "client-confidential-symmetric",
+        "client-confidential-asymmetric",
+        "context-passthrough-banner",
+        "context-passthrough-style",
+        "context-ehr-patient",
+        "context-ehr-encounter",
+        "context-standalone-patient",
+        "context-standalone-encounter",
+        "permission-offline",
+        "permission-patient",
+        "permission-user",
+        "permission-v2",
+        "authorize-post"
+    ]
+    ```
+
+3. Run the project.
+
+    ```ballerina
+    bal run
+    ```
+
+4. Invoke the APIs.
+
+    Sample request for FHIR Capability Statement:
+
+    ```
+    curl --location 'localhost:9090/fhir/r4/.well-known/smart-configuration'
+    ```
+
+
+## [Optional] Deploy in Choreo
+
+WSO2’s Choreo (https://wso2.com/choreo/) is an internal developer platform that redefines how you create digital experiences. Choreo empowers you to seamlessly design, develop, deploy, and govern your cloud native applications, unlocking innovation while reducing time-to-market. You can deploy the healthcare prebuilt services in Choreo as explained below. 
+
+### Prerequisites
+
+If you are signing in to the Choreo Console for the first time, create an organization as follows:
+
+1. Go to https://console.choreo.dev/, and sign in using your preferred method.
+2. Enter a unique organization name. For example, Stark Industries.
+3. Read and accept the privacy policy and terms of use.
+4. Click Create.
+This creates the organization and opens the Project Home page of the default project created for you.
+
+### Steps to Deploy SMART Configuration Prebuilt Service in Choreo
+1. Create Service Component
+    * Fork the pre-built Ballerina services repository (https://github.com/wso2/open-healthcare-prebuilt-services) to your Github organization.
+    * Follow the official documentation to create and configure a service: https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-1-create-a-service-component.
+
+    * Once the component creation is complete, you will see the component overview page.
+
+2. Configure and Deploy
+
+    Follow the official documentation to deploy the smart configuration prebuilt service to your organization https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-2-build-and-deploy.
+
+    On deployment configurables mentioned in [Configurations](#configurations) needs to be configured in Choreo configurable editor.
+
+### Configurations
```

Comment:
```suggestion
## Configurations
```

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/8c3081840f2925a7fbb0c1f2b97e2f9520a74af6

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/3c45d53b694704102f3950d7f4431d600fc9e72e